cli.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+./packages/cli/dist/bin/index.js $@
```

Comment:
Is this file indispensable？

Reply:
The permission of ./packages/cli/dist/bin/index.js is required although write in this script, so IMO not need write in this way:(